shell/packages/sandstorm-db/db.js

```diff
@@ -130,6 +130,8 @@ Meteor.users.ensureIndexOnServer("loginIdentities.id", { unique: 1, sparse: 1 })
 Meteor.users.ensureIndexOnServer("nonloginIdentities.id", { sparse: 1 });
 Meteor.users.ensureIndexOnServer("services.google.id", { unique: 1, sparse: 1 });
 Meteor.users.ensureIndexOnServer("services.github.id", { unique: 1, sparse: 1 });
+Meteor.users.ensureIndexOnServer("apiTokens.grainId", { sparse: 1 });
```

Comment:
Did you mean:

    ApiTokens.ensureIndexOnServer("grainId", { sparse: 1 });

?